### PR TITLE
Switch to x-eithert

### PR DESCRIPTION
**Pull request by jystic**
I upgraded all the ambiata dependencies while I was at it.

### CODE REVIEW

File: src/Jetski.hs [hunk 1]
```diff
@@ -52,7 +52,8 @@ import           System.IO.Temp (createTempDirectory)
 import           System.Posix.DynamicLinker (DL(..), RTLDFlags(..), dlopen, dlclose, dlsym)
 import           System.Process (CreateProcess(..), proc, readCreateProcessWithExitCode)
 
-import           X.Control.Monad.Catch (bracketEitherT')
+import           X.Control.Monad.Trans.Either (EitherT, pattern EitherT)
```
Comment (nhibberd) [inline src/Jetski.hs]: Why do you need the pattern here as well?

Comment (nhibberd) [inline src/Jetski.hs]: nvm im a noob

**Comment by tranma** [comment]
looks ok, does ice need to be upgraded as well? :+1:

**Comment by jystic** [comment]
@tranma Yep that's the end goal, I want to use [consume](https://github.com/ambiata/twine/blob/master/src/Twine/Parallel.hs#L116) from twine so I need to upgrade jetski+icicle+ice

**Comment by nhibberd** [comment]
:facepunch:

**Pull request merged**